app/vmagent/remotewrite/remotewrite.go

```diff
@@ -34,6 +34,8 @@ import (
 )
 
 var (
+	remoteWriteConfigCheckInterval = flag.Duration("remoteWrite.configCheckInterval", 0, "Interval for checking for changes in all remote write config files. "+
```

Comment:
It would be great to explicitly list flags which will be reloaded. That would make it easier for users to find it via search.